src/open_inwoner/accounts/views/contactmoments.py

```diff
@@ -220,16 +220,16 @@ def get_context_data(self, **kwargs):
             local_kcm.save()
 
         ctx["question"] = question
-        ctx["zaak"] = zaak.zaak if zaak else None
+        ctx["zaak"] = zaak_with_api_group.zaak if zaak_with_api_group else None
```

Comment:
Might be more readable:

```suggestion
        ctx["zaak"] = getattr(zaak_with_api_group, 'zaak', None)
```